src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -52,8 +52,8 @@ const trackDocumentationClicked = (org: Organization) =>
     user_id: ConfigStore.get('user').id,
   });
 
-function getQueryStatus(status: any) {
-  return ['open', 'closed', 'all'].includes(status) ? status : DEFAULT_QUERY_STATUS;
+function getQueryStatus(status: any): 'open' | 'closed' {
```

Comment:
a little gross having this typed as any, but I see why we're doing it, the usage of `[...].includes` on a non-any type is frustrating.